### PR TITLE
Update pr.yaml and add Unity 2020 Pipeline 

### PR DESCRIPTION
**Pull request by CDiaz-MS**
## Overview

- Modified the pr.yaml file to include Unity 2018, 2019 ~~and 2020 validation~~ 
  - Removed 2020 validation from pr.yaml due to the failing Base Cursor Test for now 
- Renamed the pr-2019.yaml file to pr-2020.yaml and added 2020 variables

### CODE REVIEW

File: pipelines/pr.yaml [hunk 1]
```diff
@@ -38,4 +38,38 @@ jobs:
...
+    - VS2019 -equals TRUE
+  steps:
+  - template: templates/common.yml
+    parameters:
+      buildUWPX86: false
+      buildUWPArm: false
+      buildUWPDotNet: false
+      UnityVersion: $(Unity2019Version)
+
+  - job: Unity2020Validation
```
Comment (keveleigh) [inline pipelines/pr.yaml:59]: I think there's a slight whitespace discrepancy here

```suggestion
- job: Unity2020Validation
```

File: pipelines/pr.yaml [hunk 2]
```diff
@@ -38,4 +38,38 @@ jobs:
...
+    - COG-UnityCache-WUS2-01
+    - SDK_18362 -equals TRUE
+    - VS2019 -equals TRUE
+  steps:
+  - template: templates/common.yml
+    parameters:
+      buildUWPX86: false
+      buildUWPArm: false
+      buildUWPDotNet: false
+      UnityVersion: $(Unity2019Version)
```
Comment (keveleigh) [inline pipelines/pr.yaml:57]: @CDiaz-MS Do we want to get just the 2019 change in for now, if we're not able to get 2020 working quite yet?

Comment (CDiaz-MS) [reply pipelines/pr.yaml:57]: Yes we can do that, I'll remove the 2020 validation from this file for now.

**Review by keveleigh — APPROVED** [review]

**Pull request merged**